tests/attrs_test.py

```diff
@@ -922,5 +923,386 @@ def f_ref(x, y, z, w):
                         check_dtypes=False)
 
 
+class BoxTest(jtu.JaxTestCase):
```

Comment:
How are boxes supposed to work under vmap or shard_map?

Reply:
Good question! The short answer is the same as mutable arrays. And Lists correspond to accum arrays, hence being amenable to parallelization like shmap/vmap.